### DISCUSSION TITLE
Google Cloud Monitoring - Add support for MQL Variable Queries

### DISCUSSION THREAD

**Discussion by sunshinekitty** [grafana/grafana#37657] [Plugins]
In the Stackdriver plugin it's not currently possible to use MQL queries to generate dashboard variables.

I'm wanting to filter one variable based off the setting of another dashboard variable.  This is not currently possible to my knowledge with the existing plugin, adding support for MQL would be my preferred way to enable this capability.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
